Backend/DataAccessLayer/Migrations/201903122358472_pending-migration.cs

```diff
@@ -0,0 +1,20 @@
+namespace DataAccessLayer.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class pendingmigration : DbMigration
+    {
+        public override void Up()
+        {
+            DropIndex("dbo.Sessions", new[] { "UserId" });
+            CreateIndex("dbo.Sessions", "UserId");
+        }
+        
+        public override void Down()
+        {
+            DropIndex("dbo.Sessions", new[] { "UserId" });
+            CreateIndex("dbo.Sessions", "UserId");
+        }
```

Comment:
The reason this "fake" migration is needed is because you modified an existing migration.

It's really best to add a new migration that updates that table.

If you can't find a way (as we discussed), the best approach is probably to remove these changes and just create a new migration that drops the existing table, and creates a new table with the correct schema.